engo_glfw.go

```diff
@@ -219,18 +225,44 @@ func RunPreparation(defaultScene Scene) {
 	SetScene(defaultScene, false)
 }
 
+func closeEvent() {
+	log.Println("Got close event")
+	for _ , scenes := range scenes {
+		scenes.scene.Exit()
+	}
+
+	if defaultCloseAction {
+		log.Println("Closing")
+		Exit()
+	} else {
+		log.Println("Warning: default close action set to false, please make sure you manually handle this")
+	}
+}
+
 func runLoop(defaultScene Scene, headless bool) {
+	c := make(chan os.Signal, 1)
```

Comment:
That looks like two tabs for me?

And there are a few other places where brackets are off (Look at your changes to the demos)